doc/varnish-configuration.rst

```diff
@@ -86,6 +98,25 @@ To enable this feature, add the following to ``your_varnish.vcl``:
 Read more on `handling PURGE requests`_ in the Varnish documentation (for
 Varnish 3, see `purging for Varnish 3`_).
 
+Purge with xkey
+^^^^^^^^^^^^^^^
+
+Alternatively for use with xkey this allows purges by URL as well as by ``key``
+header
+
+Subroutines are provided in ``resources/config/varnish-[version]/fos_purge_xkey.vcl``.
```

Comment:
i think we did a bad choice with including the ban tags in the normal ban vcl. i'd prefer to have a separate file for xkey, maybe fos_tags_xkey.vcl ?

we would then use a special method PURGEKEYS resp. SOFTPURGEKEYS|EXPIREKEYS to avoid collisions with regular PURGE.